carto/import_api.py

```diff
@@ -113,6 +116,25 @@ def __init__(self, url, client, api_version='v1', interval=None, **kwargs):
         super(URLImport, self).__init__(client, **kwargs)
 
 
+class CartoExportJob(ImportJob):
+    def __init__(self, client, visualization_id, api_key, api_version='v1', **kwargs):
+        self.viz_id = visualization_id
+        self.api_key = client.api_key
+
+        super(CartoExportJob, self).__init__(client, **kwargs)
+
+    def run(self, **import_params):
+        import_params["visualization_id"] = self.viz_id
+        self.send(EXPORT_API_URL, url_params=import_params, client_params={"http_method": "POST"})
+        self.send(EXPORT_API_URL+self.id, client_params = {"http_method": "GET"})
```

Comment:
try not to concatenate strings with `+` but, rather, use `.format`. it's kind of unimportant here, but it's good python programming practice. in any case, PEP8 must be respected (missing spaces around `+`)